pipelines/youtube_data/__init__.py

```diff
@@ -0,0 +1,181 @@
+import datetime
+from typing import Any, Dict, Iterator, List, Sequence
+
+from googleapiclient.discovery import build
+
+import dlt
+from dlt.extract.source import DltResource, DltSource
+from dlt.common.typing import TDataItem
+
+
+_SERVICE_NAME = "youtube"
+_SERVICE_VERSION = "v3"
+current_time = datetime.datetime.now()
+
+def _get_channel_id(youtube: Any, channel_name: str) -> List[str]:
+    """
+    function to retrieve channel id given channel custom name e.g. MrBeast
```

Comment:
Some style improvement hint: with some small exceptions, the most common way is to start the docstring with a capital letter.